solidity/contracts/hooks/ArbL2ToL1Hook.sol

```diff
@@ -50,30 +52,41 @@ contract ArbL2ToL1Hook is AbstractMessageIdAuthHook {
         uint32 _destinationDomain,
         bytes32 _ism,
         address _arbSys,
-        uint256 _gasQuote
+        address _childHook
     ) AbstractMessageIdAuthHook(_mailbox, _destinationDomain, _ism) {
         arbSys = ArbSys(_arbSys);
-        GAS_QUOTE = _gasQuote;
+        childHook = AbstractPostDispatchHook(_childHook);
```

Comment:
nit: if the type implies "hook" the name does not need to include this

Reply:
Better to over-specify than under-specify no?

Reply:
with naming I dont agree with this but dont feel strongly enough to debate